### PR TITLE
Use events query instead of custom call query

### LINKED ISSUE #281: We can probably remove the DAL functions for event wrapper functions
For example we have

```go
func (c *ConsoleService) GetCalls(ctx context.Context, req *connect.Request[pbconsole.GetCallsRequest])

func (c *ConsoleService) GetRequestCalls(ctx context.Context, req *connect.Request[pbconsole.GetRequestCallsRequest]) 
```

### PR DESCRIPTION
**Pull request by wesbillman**
Fixes #281

### CODE REVIEW

File: backend/controller/console.go [hunk 1]
```diff
@@ -74,16 +75,21 @@ func (c *ConsoleService) GetModules(ctx context.Context, req *connect.Request[pb
...
 
 func (c *ConsoleService) GetCalls(ctx context.Context, req *connect.Request[pbconsole.GetCallsRequest]) (*connect.Response[pbconsole.GetCallsResponse], error) {
-	calls, err := c.dal.GetModuleCalls(ctx, []string{req.Msg.Module})
+	events, err := c.dal.QueryEvents(ctx, time.Time{}, time.Now(), dal.FilterCall(types.None[string](), req.Msg.Module))
 	if err != nil {
 		return nil, errors.WithStack(err)
 	}
 
+	callEvents := filterCallEvents(events)
+	calls, err := slices.MapErr(callEvents, func(in *dal.CallEvent) (*pbconsole.Call, error) {
```
Comment (alecthomas) [inline backend/controller/console.go:84]: No error is returned, this can just be `slicse.Map()`

File: backend/controller/console.go [hunk 2]
```diff
@@ -92,13 +98,22 @@ func (c *ConsoleService) GetRequestCalls(ctx context.Context, req *connect.Reque
...
-	calls, err := c.dal.GetRequestCalls(ctx, requestKey)
+
+	events, err := c.dal.QueryEvents(ctx, time.Time{}, time.Now(), dal.FilterRequests(requestKey))
 	if err != nil {
 		return nil, errors.WithStack(err)
 	}
 
+	callEvents := filterCallEvents(events)
+	calls, err := slices.MapErr(callEvents, func(in *dal.CallEvent) (*pbconsole.Call, error) {
+		call := callEventToCall(*in)
```
Comment (alecthomas) [inline backend/controller/console.go:109]: Same.

File: backend/controller/console.go [hunk 3]
```diff
@@ -228,11 +246,11 @@ func (c *ConsoleService) StreamLogs(ctx context.Context, req *connect.Request[pb
 	}
 }
 
-func convertModuleCalls(calls []dal.CallEvent) []*pbconsole.Call {
+func convertModuleCalls(calls []dal.CallEvent) []pbconsole.Call {
```
Comment (alecthomas) [inline backend/controller/console.go:249]: These changes don't make any sense. Protobuf always uses pointers, so this just forces the caller to convert them back again?

Comment (wesbillman) [reply backend/controller/console.go:249]: Lemme see if I can make it work the other way. I did this to "fix" my new calls but it felt odd. I might ping ya with the error so you can show me the way :)

**Pull request merged**